satellite/satellitedb/repairqueue.go

```diff
@@ -117,3 +117,22 @@ func (r *repairQueue) SelectN(ctx context.Context, limit int) (segs []pb.Injured
 	}
 	return segs, Error.Wrap(rows.Err())
 }
+
```

Comment:
add a comment here. Make sure to clarify that this counts every segment, regardless of how recently it was attempted

Reply:
done